website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -328,7 +328,8 @@ builder.
 
 - `vpc_id` (string) - If launching into a VPC subnet, Packer needs the VPC ID
   in order to create a temporary security group within the VPC. Requires `subnet_id`
-  to be set.
```

Comment:
I know, this is a bear, but this change needs to be applied to all the docs for each of the builders. 

* website/source/docs/builders/amazon-ebssurrogate.html.md
* website/source/docs/builders/amazon-ebsvolume.html.md
* website/source/docs/builders/amazon-instance.html.md